frontend/src/components/rider/common/viewRequest.js

```diff
@@ -49,9 +56,20 @@ function ViewRequest() {
         })             
     }
 
-    return (
+    return (     
```

Comment:
should've wrote something like:-
`if(loading) {
return <LoadingScreen />
}
return (
...rest of the code)` 
rather than using a conditional operator right in the starting.